File: codes/math_LCM.py
def GCD (a, b):
	if b == 0:
		return a
	else:
		return GCD(b, a%b)

def LCM (a, b):
	return (a*b) / GCD(abs(a),abs(b))

a, b = input(), input()
print (LCM(a,b))

File: codes/math_LCM.cpp
// Euclid's Least Common Multiple Algorithm
#include <stdio.h>
#include <iostream>
#include <math.h>

int GCD (int a, int b)
{
	return b == 0 ? a : GCD(b,a%b);
}

int LCM (int a, int b)
{
	return (a * b) / GCD(fabs(a),fabs(b));
}

int main ()
{
	int a,b,lcm;

	scanf ("%d %d", &a, &b);

	lcm = LCM (a,b);
	printf ("%d\n", lcm); 
}
